mappings/net/minecraft/block/Block.mapping

```diff
@@ -17,8 +26,8 @@ CLASS net/minecraft/class_18 net/minecraft/block/Block
 	FIELD field_2091 RAIL Lnet/minecraft/class_18;
 	FIELD field_211 GRASS Lnet/minecraft/class_24;
 	FIELD field_213 BEDROCK Lnet/minecraft/class_18;
-	FIELD field_214 WATER Lnet/minecraft/class_18;
-	FIELD field_216 LAVA Lnet/minecraft/class_18;
+	FIELD field_214 WATER_MOVING Lnet/minecraft/class_18;
+	FIELD field_216 LAVA_MOVING Lnet/minecraft/class_18;
```

Comment:
I'd only really suggest `X_FLOWING` if this needs to be changed.
Moving is technically correct here, tho probably not the first word to come to mind for most when it comes to fluids.

Reply:
Changed this

Reply:
should probably also be `FLOWING_WATER` rather than `WATER_FLOWING` so as to follow the natural word order

Reply:
That could be good, but if someone is typing water into their IDE, they may not like the switch as much because the auto correct may not find it as easily. But up to you.

Reply:
natural word order is the convention in yarn, which these mappings are ultimately forked from, so I would stick to that for now (as lots of other mappings would need updating too if the convention were to change)

Reply:
Done